pyramda/reject.py

```diff
@@ -0,0 +1,6 @@
+from pyramda.function.curry import curry
+from pyramda.iterable.filter import filter
+
+
+reject = curry(lambda predicate, filterable: filter(
```

Comment:
There is about a handful of other methods that could be defined too, for the sake consistency. Maybe put that as an issue, to be resolved.